src/Colors/index.js

```diff
@@ -41,6 +45,10 @@ const colors = {
 
   brandDestructive: "#EB5A47",
   brandAffirmative: "#61BD4F",
-  brandWarning: "#FFBD00"
+  brandWarning: "#FFBD00",
+
+  brandPrimary: "#03D4BF",
```

Comment:
hey good job in general. 👍 remove this one it is already existing in the line 34.